ConsentViewController/Classes/SourcePointClient/ConsentStatusMetadata.swift

```diff
@@ -13,9 +13,48 @@ struct ConsentStatusMetaData: QueryParamEncodable {
         let applies: Bool
         let dateCreated: SPDate?
         let uuid: String?
-        let hasLocalData: Bool
+        let hasLocalData = false
```

Comment:
do we need to set a default?

Reply:
It was always hardcoded `false`, but in another point of the code. In this line, I hardcode directly in the class, so other parts of the code don't have to set it to `false`.